mne/tests/test_epochs.py

```diff
@@ -927,6 +928,10 @@ def test_access_by_name():
     assert_array_equal(epochs.events, epochs6.events)
     assert_array_almost_equal(epochs.get_data(), epochs6.get_data(), 20)
 
+    # Make sure we preserve names
+    assert_equal(epochs['a'].name, 'a')
+    assert_equal(epochs[['a', 'b']]['a'].name, 'a')
+
```

Comment:
Mmmh, not sure where these changes are coming from. I'll remove these changes at the next push